store/generated_key/eigenda/verify/verifier.go

```diff
@@ -30,6 +30,7 @@ type Config struct {
 	// below fields are only required if VerifyCerts is true
 	RPCURL               string
 	SvcManagerAddr       string
+	CertVerifierV1Addr   string // used if enabling custom quorums/thresholds
```

Comment:
```suggestion
	CertVerifierV1Addr   string // used to enable custom quorums/thresholds
```

Reply:
its not "if" right? Since there is no boolean controlling whether those are used.